### ISSUE TITLE
i18n and l10n the Select File button

### ISSUE THREAD

**Issue by joanise** [ReadAlongs/Studio-Web-archived#36]
Because we're using a default angular file upload button (search for two occurrences of `type="file"` in `upload.component.html`), it doesn't get i18n'd:

![image](https://user-images.githubusercontent.com/33264708/211375932-9244b376-6c7d-4d11-82ae-bcbc3b1fdf3f.png)

It looks like the solution is to completely do the button manually: https://stackoverflow.com/q/20517321/20343209 😞

**Comment by dhdaines** [comment]
Ah yeah, file upload buttons are annoying like that ... on the other hand it does get the browser's default language

**Comment by joanise** [comment]
I suppose we can consider that good enough. My own browser was keeping the buttons in English, but I just realized it's because I had Chrome in English on this machine. Just switched it, and it does indeed switch the language of the buttons.

**Comment by joanise** [comment]
I guess that's why so many web sites show me upload and some other buttons in French on my personal computer. I've noticed every now and again there's a form where the standard fields are in French but the rest of the text is in English, and I guess it's those automated Angular (and similar) fields.

I just switched my browser language to French, and I should set my computer to French too, so I see those things as I'm testing software.
